admin/onpage/class-onpage-service.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * @package WPSEO\Admin\OnPage
+ */
+
+/**
+ * Class WPSEO_OnPage_Service
+ */
+class WPSEO_OnPage_Service {
```

Comment:
I would rename this to `WPSEO_Ryte_Service` because this is new code and `OnPage.org` has been renamed to `Ryte`